docs/cases/cases-manage.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[cases-open-manage]]
```

Comment:
For some reason, the mini-nav on this page only includes H2 levels, but in the current live docs they also have H3s. This seems worth troubleshooting, because there are a lot of action-based sub-sections here that would be helpful to see in the mini-nav.

PR's preview page (only includes H2s):
<img width="322" alt="image" src="https://user-images.githubusercontent.com/87339667/173621268-a77f9d08-ef25-48a0-b7d7-9a200f43eef8.png">

Current live page (includes H2s and H3s):
<img width="260" alt="image" src="https://user-images.githubusercontent.com/87339667/173621337-42aee278-2a2e-4c29-8ddb-6e3255c375bc.png">

Reply:
Yeah, I was looking into this yesterday but didn't want to delay the review any further so I published it as is. I'm still not sure why the H3s aren't showing up so I'm going to need to poke around a bit more on this. If anyone has faced this and fixed it before, please share your tips!

Reply:
I believe it's related to how the various index pages, `includes`, and `leveloffset` settings are interacting. It'll take some tinkering and maybe re-architecting to get those sub-topics to appear in the mini-nav, but I see it as somewhat lower priority "nice to have" at this point, and definitely not something to delay the rest of the changes in this PR.

Reply:
Thanks for investigating this @joepeeples ! Your notes match the observations I've made and I'm in full agreement for re-architecting the index files after the release. I still haven't figured out how the cases index file is interacting with the main and mini TOC and it'll take a lot more tweaking and testing than we have time for atm. After this release, I think it'd be worth it to consolidate the cases and Timeline index files so we can enforce the same hierarchy structure across the topics.